include/version.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
```

Comment:
Can you update the include guards in this one as well?

On a related note, should we add a comment to this file saying it will be replaced at configure-time with the proper version, in case anyone looks at it and gets confused?

Reply:
Done